cgroups/fs/apply_raw.go

```diff
@@ -128,14 +128,23 @@ func (m *Manager) Apply(pid int) error {
 }
 
 func (m *Manager) Destroy() error {
-	return cgroups.RemovePaths(m.Paths)
+	m.Lock()
+	defer m.Unlock()
+	if err := cgroups.RemovePaths(m.Paths); err != nil {
+		return err
+	}
+	return nil
 }
 
 func (m *Manager) GetPaths() map[string]string {
+	m.Lock()
+	defer m.Unlock()
```

Comment:
Ok it was like this but then we said something about locking for the whole GetStats (maybe in Docker not here)